### PR TITLE
K8SPXC-394 fix 'security-context' for openshift

### PR DESCRIPTION
**Pull request by hors**
[K8SPXC-394](https://jira.percona.com/browse/K8SPXC-394)

### CODE REVIEW

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| recreate | passed |
| big-data | passed |
| storage | passed |
| scaling | passed |
| init-deploy | passed |
| self-healing | passed |
| scaling-proxysql | passed |
| limits | passed |
| restore-to-encrypted-cluster | passed |
| upgrade | passed |
| monitoring | passed |
| self-healing-advanced | passed |
| upgrade-consistency | passed |
| security-context | passed |
| demand-backup | passed |
| operator-self-healing | passed |
| monitoring-2-0 | passed |
| affinity | passed |
| scheduled-backup | failed |
| one-pod | passed |
| auto-tuning | passed |
| proxysql-sidecar-res-limits | passed |
| users | passed |
| demand-backup-encrypted-with-tls | failed |

File: e2e-tests/security-context/run [hunk 1]
```diff
@@ -11,9 +11,14 @@ deploy_cert_manager
 
 kubectl_bin apply -f "$test_dir/conf/service-account.yml"
 if [[ "${OPENSHIFT}" == 1 ]]; then
-    oc adm policy add-scc-to-user privileged -z percona-xtradb-cluster-operator ${OPERATOR_NS:+-n $OPERATOR_NS}
+    oc adm policy add-scc-to-user privileged -z default
```
Comment (delgod) [inline e2e-tests/security-context/run:14]: please remove if this line is not needed

File: e2e-tests/security-context/run [hunk 2]
```diff
@@ -73,7 +78,7 @@ wait_backup_restore $restore
 compare_kubectl job.batch/restore-job-$restore-$cluster
 
 if [[ "${OPENSHIFT}" == 1 ]]; then
-    oc adm policy remove-scc-from-user privileged -z percona-xtradb-cluster-operator ${OPERATOR_NS:+-n $OPERATOR_NS}
+    oc adm policy remove-scc-from-user privileged -z default
```
Comment (delgod) [inline e2e-tests/security-context/run:81]: please remove if this line is not needed

**Review by delgod — CHANGES_REQUESTED** [review]

File: e2e-tests/security-context/run (hunk 1 above)
Comment (hors) [reply e2e-tests/security-context/run]: done

File: e2e-tests/security-context/run (hunk 2 above)
Comment (hors) [reply e2e-tests/security-context/run]: done

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| init-deploy | passed |
| scaling | passed |
| storage | passed |
| big-data | passed |
| recreate | passed |
| self-healing | passed |
| limits | passed |
| monitoring | passed |
| restore-to-encrypted-cluster | passed |
| self-healing-advanced | passed |
| scaling-proxysql | passed |
| upgrade | passed |
| monitoring-2-0 | passed |
| demand-backup | passed |
| affinity | passed |
| upgrade-consistency | passed |
| one-pod | passed |
| security-context | passed |
| scheduled-backup | failed |
| operator-self-healing | passed |
| auto-tuning | passed |
| proxysql-sidecar-res-limits | passed |
| users | passed |
| demand-backup-encrypted-with-tls | failed |

**Review by delgod — APPROVED** [review]

**Review by cap1984 — APPROVED** [review]

**Pull request merged**